llvm/lib/Support/SourceMgr.cpp

```diff
@@ -188,6 +189,29 @@ SourceMgr::SrcBuffer::~SrcBuffer() {
   }
 }
 
+StringRef SourceMgr::getDebugFilename(SMLoc Loc, unsigned BufferID) const {
+  if (!BufferID)
+    BufferID = FindBufferContainingLoc(Loc);
+  assert(BufferID && "Invalid location!");
+
+  auto &SB = getBufferInfo(BufferID);
+
+  return SB.SourceFilename;
+}
+
+std::pair<unsigned, unsigned>
+SourceMgr::getDebugLineAndColumn(SMLoc Loc, unsigned BufferID) const {
+  if (!BufferID)
+    BufferID = FindBufferContainingLoc(Loc);
+  assert(BufferID && "Invalid location!");
+
+  auto &SB = getBufferInfo(BufferID);
+
```

Comment:
delete blank line.

We generally don't force a blank line after a variable declaration.